src/Sylius/Bundle/ShopBundle/Resources/views/Product/Show/_price.html.twig

```diff
@@ -1,5 +1,8 @@
 {% import "@SyliusShop/Common/Macro/money.html.twig" as money %}
 
+<span class="ui small header" style="text-decoration: line-through" id="product-original-price">
```

Comment:
Can we determine if we even want to add this html block?

Reply:
You mean in case original price is null or higher than the price, we should not print this HTML?
Currently this login is hold be a twig macro.

Reply:
I think this condition should be put here - we don't need a redundant html with no text in it :)

Reply:
```
{% set originalPrice = money.calculateOriginalPrice(product|sylius_resolve_variant) %}
{% if originalPrice is not null %}
    <span>{{ originalPrice }} </span>
{% endif %}
```

Should do the work